cfn-resources/access-list-api-key/mongodb-atlas-accesslistapikey.json

```diff
@@ -1,70 +1,73 @@
 {
-    "typeName": "MongoDB::Atlas::AccessListAPIKey",
-    "description": "Creates the access list entries for the specified organization API key.",
-    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
-    "definitions": {
+  "typeName": "MongoDB::Atlas::AccessListAPIKey",
+  "description": "Creates the access list entries for the specified organization API key.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
```

Comment:
Looking into Aastha comment in jira card (https://jira.mongodb.org/browse/INTMDB-1161?focusedCommentId=5741225&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-5741225) we can modify the sourceUrl property in these PRs. What do you think?

Reply:
done!